components/TextFields/src/MDCTextInputControllerDefault.m

```diff
@@ -653,6 +670,18 @@ - (UIOffset)floatingPlaceholderOffset {
   return UIOffsetMake(horizontal, vertical);
 }
 
+- (CGFloat)floatingPlaceholderAnimationConstraintLeadingConstant:(UIEdgeInsets)textInsets
```

Comment:
If the UIEdgeInsets don't need to be swapped for RTL, could you please document that you expect them to arrived with leading mapped to left, and trailing mapped to right? It would reduce my RTL-related anxiety. :)

Reply:
Done!